### PR TITLE
Ai aim mod config

### PR DESCRIPTION
**Pull request by bmorel**
This allows server owners to modify per faction aim speed modifier, to adapt the overall AI balancing to their taste.

### CODE REVIEW

**Comment by bmorel** [comment]
this should help server owners working around #2439

File: src/sgame/sg_bot_util.cpp [hunk 1]
```diff
@@ -35,6 +35,8 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 #include <glm/gtx/vector_angle.hpp>
 
 static Cvar::Range<Cvar::Cvar<int>> g_bot_defaultSkill( "g_bot_defaultSkill", "Default skill value bots will have when added", Cvar::NONE, 5, 1, 9 );
+static Cvar::Cvar<int> g_bot_alienAimModifier = Cvar::Cvar<int>( "g_bot_alienAimModifier", "make bots of alien team slower to aim", Cvar::NONE, 250 );
```
Comment (slipher) [inline src/sgame/sg_bot_util.cpp:38]: I suggest naming it `g_bot_alienAimDelay`. With "modifier" it's unclear whether a higher number means better or worse aiming. Description could be like "alien bots' aiming time in milliseconds"

**Review by slipher — COMMENTED** [review]
I verified that the RML looks good

File: src/sgame/sg_bot_util.cpp (hunk 1 above)
Comment (bmorel) [reply src/sgame/sg_bot_util.cpp]: done

**Comment by bmorel** [comment]
updated the variable name as asked. Anything else?

**Comment by slipher** [comment]
LGTM

**Pull request merged**